languages/translations/de.json

```diff
@@ -84,12 +84,12 @@
         },
         "continue": {
             "default": {
-                "default": "{modifier} weiterfahren",
+                "default": "{modifier} abbiegen",
                 "name": "{modifier} weiterfahren auf {way_name}",
-                "destination": "{modifier} weiterfahren Richtung {destination}"
+                "destination": "{modifier} abbiegen Richtung {destination}"
```

Comment:
Why did we change from "weiterfahren" (continue) here to "abbiegen" (turn) for the `continue` instruction?

Reply:
So, most of the `continue` maneuvers require you to make a turn so we changed the `default` to use `turn` and then created an override for `straight` to say continue. I'm guess that the translator say this change in english and made it here, but didn't provide straight override.